examples/pipeline_wavernn/datasets.py

```diff
@@ -0,0 +1,78 @@
+import os
+import random
+import torch
+import torchaudio
+from torchaudio.datasets import LJSPEECH
+
+
+class ProcessedLJSPEECH(LJSPEECH):
+
+    def __init__(self,
+                 files,
+                 transforms,
+                 mode,
+                 n_bits):
+
+        self.transforms = transforms
+        self.files = files
+        self.mode = mode
+        self.n_bits = n_bits
+
+    def __getitem__(self, index):
+
+        file = self.files[index]
+        x, sample_rate = torchaudio.load(file)
+        mel = self.transforms(x)
+
+        bits = 16 if self.mode == 'MOL' else self.n_bits
+
+        x = (x + 1.) * (2 ** bits - 1) / 2
+        x = torch.clamp(x, min=0, max=2 ** bits - 1)
+
+        return mel.squeeze(0), x.int().squeeze(0)
```

Comment:
This converts representation of a waveform from [-1, 1] to 16-bit integer representation. For instance, this is done in [load_wav](https://github.com/pytorch/audio/blob/master/torchaudio/backend/sox_backend.py#L68) already. Since this is an important step and can be generalized, let's make this into a function within torchaudio. One point of discussion is whether we add that directly in WaveRNN.

Reply:
This function has been added as `normalized_waveform_to_bits` function in `processing.py`.